### PR TITLE
feat: 使用 Azure OpenAI 进行翻译

### PR DESCRIPTION
**Pull request by dogdie233**

### CODE REVIEW

**Review by unlsycn — CHANGES_REQUESTED** [review]

**Review by unlsycn — DISMISSED** [review]

File: TranslatorLibrary/AzureOpenAITranslator.cs [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+                Stream = false
+            };
+        }
+
+        #region DataModel
+
+        public class ChatRequest
+        {
+            [JsonPropertyName("messages")] public List<MessageRequest> Messages { get; set; } = new();
+            [JsonPropertyName("stream")] public bool Stream { get; set; } = false
```
Comment (unlsycn) [inline TranslatorLibrary/AzureOpenAITranslator.cs:141]:
```suggestion
            [JsonPropertyName("stream")] public bool Stream { get; set; } = false;
```

**Review by unlsycn — CHANGES_REQUESTED** [review]

File: TranslatorLibrary/AzureOpenAITranslator.cs (hunk 1 above)
Comment (dogdie233) [reply TranslatorLibrary/AzureOpenAITranslator.cs:141]: its my stupid fault😓😓

**Comment by unlsycn** [comment]
Thanks for your contirbution! Please squash for tidier git history

**Comment by unlsycn** [comment]
Thanks for your contirbution!

**Pull request merged**